gallery/visualtests/nxCollapsibleMultiSelect.js

```diff
@@ -10,9 +10,15 @@ describe('NxCollapsibleMultiSelect', function() {
   const { focusTest, simpleTest, a11yTest } = setupBrowser('#/pages/Collapsible Multi-Select');
 
   const selector = '#nx-collapsible-multi-select-example .nx-collapsible-items--select';
+  const disabledWithTooltipSelector
+    = '#nx-collapsible-multi-select-disabled-with-tooltip-example .gallery-example-live';
 
   it('looks right', simpleTest(selector));
 
+  describe('NxCollapsibleMultiSelect Disabled With Tooltip', function() {
+    it('looks right', simpleTest(disabledWithTooltipSelector));
```

Comment:
Should probably have a hover+tooltip test as well given that the initial fix in this PR didn't address the tooltip correctly.